### DISCUSSION TITLE
7777/07/07/diary/

### DISCUSSION THREAD

**Discussion by giscus** [qllokirin/hexo_blog_comments#1] [Announcements]
# 7777/07/07/diary/

https://70lokirin.github.io/7777/07/07/diary/

**Comment by qllokirin** [comment]
haihaihai

**Comment by qllokirin** [comment]
`du -sh *`查看当前目录下文件夹大小 非常好用

**Comment by zzsqwq** [comment]
相机延迟问题可能是队列缓存区大小的问题？

**Reply by qllokirin** [reply]
哈哈，战sir似乎没有发现我把日记调转过来了，最上面是最新的，最下面是最旧的，嘿嘿

**Comment by Charile0304** [comment]
第一次看到这篇日记，忽然发现稀稀疏疏平平淡淡的日常其实才是组成我们生活的主要部分，而记录则让这部分有了温度与分量，心生感慨。只顾着埋头走路的我也该认真思考是不是应该写一写属于我自己的日记了。😌😌😌
